tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/domain/model/notion/EmployeeInfoEntity.kt

```diff
@@ -0,0 +1,71 @@
+package band.effective.office.tv.domain.model.notion
+
+import band.effective.office.tv.utils.DateUtlils
+import java.util.*
+
+class EmployeeInfoEntity(
+    val firstName: String,
+    val startDate: String,
+    val nextBirthdayDate: String,
+    val photoUrl: String,
+)
+
+fun List<EmployeeInfoEntity>.processEmployeeInfo(): MutableList<EmployeeInfo> {
+    val resultList = mutableListOf<EmployeeInfo>()
```

Comment:
Also problem here. 

`map()` returns a list with already changed value, so you can do your checking ( `isCelebrationToday` etc ). 

In another way: if story with `celebration day` and `newEmployeeToday` exists then it make sense to create `resultList`, but do checking of employee by `forEach` , `map` means some reformatting on entities